Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/ElasticSearchClient.php

```diff
@@ -34,12 +35,26 @@ class ElasticSearchClient extends \Flowpack\ElasticSearch\Domain\Model\Client
     protected $indexName;
 
     /**
+     * The language suffix
+     *
+     * @var string
+     */
+    protected $languageDimension;
```

Comment:
Dimensions are not only about languages, although they mostly used for that purpose. We need a solution that does not equate dimension-presets with a language.